Memeolist/Memeolist/view/comments/CommentsViewController.swift

```diff
@@ -0,0 +1,47 @@
+import AGSSync
+import UIKit
+
+class CommentsViewController: UIViewController {
+    
+    var meme: AllMemesQuery.Data.AllMeme?
+    var tableViewController: CommentsTableViewController?
+
+    @IBOutlet weak var commentTextView: UITextView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.navigationItem.title = "Comments"
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    @IBAction func addButtonClicked(_ sender: Any) {
+        guard let id = meme?.id, let owner = meme?.owner else { return }
+        
+        AgsSync.instance.client?.perform(mutation: PostCommentMutation(memeid: id, comment: commentTextView.text, owner: owner)) { result, error in
+            if let comment = result?.data?.postComment {
+                self.commentTextView.text = ""
+                self.meme?.comments.insert(AllMemesQuery.Data.AllMeme.Comment.init(id: comment.id, comment: comment.comment, owner: comment.owner), at: 0)
```

Comment:
we can actually use `Comment(id: comment.id, comment: comment.comment, owner: comment.owner)` instead of `AllMemesQuery.Data.AllMeme.Comment.init(id: comment.id, comment: comment.comment, owner: comment.owner)`